exporter/loggingexporter/logging_exporter.go

```diff
@@ -33,34 +33,21 @@ type loggingExporter struct {
 	debug  bool
 }
 
-func (s *loggingExporter) pushTraceData(
-	_ context.Context,
-	td pdata.Traces,
-) error {
-
+func (s *loggingExporter) pushTraceData(_ context.Context, td pdata.Traces) error {
 	s.logger.Info("TracesExporter", zap.Int("#spans", td.SpanCount()))
 
-	if !s.debug {
-		return nil
+	if s.debug {
+		s.logger.Debug(otlptext.Traces(td))
 	}
-
-	s.logger.Debug(otlptext.Traces(td))
-
 	return nil
 }
```

Comment:
It is more readable to read:
```
* if not precondition return
doaction
```
than
```
* if precondition doaction
return 
```

Reply:
This is not a regular logical branch. It's more canonical to write code like:

```
if verbose {
   // print here
}
```

Feel free to close it, I don't have strong opinions. It was hard for me to read it, hence created the PR.